mocha.css

```diff
@@ -36,48 +35,47 @@
 
 @media (prefers-color-scheme: dark) {
   :root {
-     --mocha-color: #fff;
-     --mocha-bg-color: #222;
-     --mocha-pass-icon-color: #00d6b2;
-     --mocha-pass-color: #222;
-     --mocha-pass-shadow-color: rgba(255,255,255,.2);
-     --mocha-pass-mediump-color: #f1be67;
-     --mocha-pass-slow-color: #f49896;
-     --mocha-test-pending-color: #0b97c4;
-     --mocha-test-pending-icon-color: #0b97c4;
-     --mocha-test-fail-color: #f44;
-     --mocha-test-fail-icon-color: #f44;
-     --mocha-test-fail-pre-color: #fff;
-     --mocha-test-fail-pre-error-color: #f44;
-     --mocha-test-html-error-color: #fff;
-     --mocha-box-shadow-color: #444;
-     --mocha-box-bottom-color: #555;
-     --mocha-test-replay-color: #fff;
-     --mocha-test-replay-bg-color: #444;
-     --mocha-stats-color: #aaa;
-     --mocha-stats-em-color: #fff;
-     --mocha-stats-hover-color: #444;
-     --mocha-progress-ring-color: #444;
-     --mocha-progress-ring-highlight-color: #888;
-     --mocha-progress-text-color: #fff;
-     --mocha-error-color: #f44;
-
-     --mocha-code-comment: #ddd;
-     --mocha-code-init: #9cc7f1;
-     --mocha-code-string: #80d4ff;
-     --mocha-code-keyword: #e3a470;
-     --mocha-code-number: #4ca7ff;
+    --mocha-color: #fff;
+    --mocha-bg-color: #222;
+    --mocha-test-pass-color: #00d6b2;
+    --mocha-test-pass-duration-color: #222;
+    --mocha-test-pass-shadow-color: rgba(255, 255, 255, 0.2);
+    --mocha-test-pass-mediump-color: #f1be67;
+    --mocha-test-pass-slow-color: #f49896;
+    --mocha-test-pending-color: #0b97c4;
+    --mocha-test-pending-icon-color: #0b97c4;
+    --mocha-test-fail-color: #f44;
+    --mocha-test-fail-icon-color: #f44;
+    --mocha-test-fail-pre-color: #fff;
+    --mocha-test-fail-pre-error-color: #f44;
+    --mocha-test-html-error-color: #fff;
+    --mocha-box-shadow-color: #444;
+    --mocha-box-bottom-color: #555;
+    --mocha-test-replay-color: #fff;
+    --mocha-test-replay-bg-color: #444;
+    --mocha-stats-color: #aaa;
+    --mocha-stats-em-color: #fff;
+    --mocha-stats-hover-color: #444;
+    --mocha-progress-ring-color: #444;
+    --mocha-progress-ring-highlight-color: #888;
+    --mocha-progress-text-color: #fff;
+    --mocha-error-color: #f44;
+    --mocha-code-comment: #ddd;
+    --mocha-code-init: #9cc7f1;
+    --mocha-code-string: #80d4ff;
+    --mocha-code-keyword: #e3a470;
+    --mocha-code-number: #4ca7ff;
   }
 }
 
 body {
-  margin:0;
+  margin: 0;
```

Comment:
Could you please revert unrelated changes? I personally prefer them the edited way (were the edits from Prettier? 😄) but it pollutes the diff a bit.

If we want to auto-format this file, it'd be good to do it in a separate standalone issue + PR. There's probably some tooling work to be done to standardize the repo's formatting.

Reply:
works for me :)